configuration/README.md

```diff
@@ -84,6 +88,9 @@ Argument propagators are identified via regexp matching the fully-qualified type
 }
 ```
 
+For matchers that accept a `ReceiverRE` regexp matcher, an unspecified string will match any (or no) receiver.
```

Comment:
Should we specify the (empty) `ReceiverRE` fields anyway in `example-config.json`? I feel like the behavior related to unspecified strings is an artifact of how Go handles JSON unmarshalling + default values, so I think it could be clearer to mention the field explicitly in the example config.

Reply:
(I would have also suggested adding comments directly into the example config, but JSON doesn't support comments :disappointed:)

Reply:
Good catch.  Added.